core/src/main/java/org/infinispan/factories/EmptyConstructorFactory.java

```diff
@@ -27,31 +23,26 @@
  * @since 4.0
  */
 
-@DefaultFactoryFor(classes = {InboundInvocationHandler.class, RemoteCommandsFactory.class, ExternalizerTable.class,
-                              BackupReceiverRepository.class, CancellationService.class, TimeService.class})
+@DefaultFactoryFor(classes = {BackupReceiverRepository.class, CancellationService.class, ExternalizerTable.class,
+                              InboundInvocationHandler.class, RemoteCommandsFactory.class, TimeService.class})
```

Comment:
Ah gotcha you were going for alphabetical order :) - I guess we should probably do that for these larger ones moving forward.